ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -1217,6 +1217,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
+- as: gcp-ipi-oidc-rt-fips-amd-regen-cert-f28
```

Comment:
I remember @wangke19 said `regen` job will take longer time then other jobs, so is it proper to move it from amd64 to multi-arch?

Reply:
moving it from amd to  mult-arch does not have impact on how long the testing will take, or less, it is a diconnected install which need to mirror a bigger payload image.

Reply:
Note that, the testing is a e2e testing, but not a upgrade + regen-cert testing.